### ISSUE TITLE
Step 4, 5 and 6 could be before step 2

### ISSUE THREAD

**Issue by aniablaziak** [foundersandcoders/express-handlebars-workshop#23]
Logically it makes more sense and we found that quite a few people were confused about what step 2 was doing without the views folder being there.

**Comment by astroash** [comment]
@aniablaziak agreed! This would be an improvement on the flow of the instructions. PRs welcome 🙂
